src/components/common/contact-form.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import ContactFormStyles from './contact-form.module.scss'
+
+export default ({ name, messageCopy }) => {
+  return (
+    <form
+      className={ContactFormStyles.contactForm}
+      name={name}
+      method="POST"
+      action="/contact/success"
+      netlify-honeypot="covid-bot-field"
+      data-netlify="true"
+    >
+      <div>
+        <input type="hidden" name="form-name" value={name} />
+        <label htmlFor="name">
+          <p className={ContactFormStyles.inputHeader}>
```

Comment:
Can we use `label` here and associate the labels with the inputs with `for`?

Reply:
Do you mean something like:

```
        <label htmlFor="email" className={ContactFormStyles.inputHeader}>
          What is your email address?
          <input type="email" id="email" name="email" required />
        </label>
```

Reply:
Yes! Perfect.

Relatedly, I'm not sure if we need the `id` attrs...is that for Netlify?

Reply:
Nope, good catch! The only thing we need for Netlify here is that hidden input, I think.